packages/ra-ui-materialui/src/button/BulkUpdateButton.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { FC } from 'react';
+import PropTypes from 'prop-types';
+import BulkUpdateWithConfirmButton, {
+    BulkUpdateWithConfirmButtonProps,
+} from './BulkUpdateWithConfirmButton';
+import BulkUpdateWithUndoButton, {
+    BulkUpdateWithUndoButtonProps,
+} from './BulkUpdateWithUndoButton';
+
+/**
+ * Updates the selected rows.
+ *
+ * To be used inside the <List bulkActionButtons> prop (where it's enabled by default).
+ *
+ * @example // basic usage
+ * import * as React from 'react';
+ * import { Fragment } from 'react';
+ * import { BulkUpdateButton, BulkExportButton } from 'react-admin';
+ *
+ * const PostBulkActionButtons = ({ basePath }) => (
+ *     <Fragment>
+ *         <BulkExportButton />
+ *         <BulkUpdateButton label="Reset Views" data={{ views: 0 }} basePath={basePath} />
+ *     </Fragment>
+ * );
+ *
+ * export const PostList = (props) => (
+ *     <List {...props} bulkActionButtons={<PostBulkActionButtons />}>
+ *         ...
+ *     </List>
+ * );
+ */
+const BulkUpdateButton: FC<BulkUpdateButtonProps> = ({ undoable, ...props }) =>
+    undoable ? (
```

Comment:
we're deprecating `undoable` in favor of `mutationMode` (see e.g. `DeleteButton`). Could you use `mutationMode` directly here?